pallets/prices/src/lib.rs

```diff
@@ -118,21 +110,24 @@ impl<T: Config> PriceFeeder for Pallet<T> {
     /// Timestamp is zero means the price is emergency price
     fn get_price(currency_id: &CurrencyId) -> Option<PriceDetail> {
         // if emergency price exists, return it, otherwise return latest price from oracle.
-        Self::get_emergency_price(currency_id).or_else(|| {
-            T::Source::get(&currency_id).and_then(|price| {
-                price
-                    .value
-                    .into_inner()
-                    .checked_div(1_000_000_000_000_000)
-                    .and_then(|r| Some((r, price.timestamp)))
-            })
-        })
+        let origin_price = Self::get_emergency_price(currency_id).or_else(|| {
+            T::Source::get(&currency_id).and_then(|price| Some((price.value, price.timestamp)))
+        });
+        if let (Some((price, timestamp)), Some(decimal)) =
+            (origin_price, 10u128.checked_pow(CURRENCY_DECIMAL))
```

Comment:
Can we put this logic at client side?

Reply:
I changed this because, after my test, I found that the conversion between Unit and Pico was not taken into account in the amount. This change also exists in ocw-oracle, so that the price stored in oracle is the same as the price of orml-oracle.
Put this logic to the client-side is ok, I think we can use this solution first after we adjust the client and then remove this logic.